src/redux/slices/graphSlice.ts

```diff
@@ -2,9 +2,20 @@ import { DSNPUserId } from "@dsnp/sdk/dist/types/core/identifiers";
 import { createSlice, PayloadAction } from "@reduxjs/toolkit";
 import { GraphChange } from "../../utilities/types";
 
-interface graphState {
-  following: Record<DSNPUserId, Record<DSNPUserId, boolean>>;
-  followers: Record<DSNPUserId, Record<DSNPUserId, boolean>>;
+export enum RelationshipStatus {
+  ESTABlISHED,
+  UPDATING,
```

Comment:
Updating doesn't store the data about if the update is an update to follow or an update to unfollow. Do we want to know which way the update is moving?

Reply:
We don't currently care. We're either following someone (established), not following (indicated by absence), or somewhere in-between (updating). This is purely for display state, and updating just means the user has changed the relationship, but we haven't yet seen the change on chain. We display both transitions in the same way.